### DISCUSSION TITLE
Thank you for your contributions in September and October 2024

### DISCUSSION THREAD

**Discussion by MisRob** [learningequality/kolibri#12812] [Community]
Hi friends!

On behalf of the Learning Equality team, I'd like to extend our gratitude to @BabyElias @GarvitSinghal47 @iamshobhraj @lokesh-sagi125 @muditchoudhary @nikkuAg @RONAK-AI647 @Sahil-Sinha-11 @shivam-daksh @Suraj-kumar00 @thesujai

…and all new contributors, as well as those whose work is still in progress or waiting to be merged.

Alongside pull requests from the final GSOC period, we merged the following updates over the past two months:

**Kolibri**

- Keyboard navigation in radio button groups is improved in Firefox in some first places
- Practice quiz cards show score
- Drop shadows are updated to align with the latest Kolibri Design System guidelines
- Vuex data store usage is replaced with the more modern Composition API in several places
- Hardcoded color is replaced by theme
- Force delete of channel resources is fixed
- Automated saving of new contributors to the authors file is fixed

**Kolibri Design System (KDS)**

- Radio button component shows a developer-only warning when used incorrectly
- Icon button component uses the tooltip value as an `aria-label` fallback when it's not specified explicitly
- Focus trap component is moved from Kolibri to KDS and used in the modal component
- Switch component can receive `aria-labelledby`
- Website favicon is updated to the new brand logo
- Table component’s sorting cycle is improved to include an unsorted state

**Miscellaneous**

- Significant progress on the automated script that saves new community pull requests to a spreadsheet

It’s wonderful to see you all continuing this wonderful and impactful work and new contributors joining. I am also pleased to see some of our GSOC contributors guiding new volunteers by opening issues, answering questions, and even working on pull request reviews.

**Thank you all for your dedication and support!**

---

**_If you haven’t already, we’d love for you to [check out the Kolibri Contributors Community](https://github.com/learningequality/kolibri/discussions/12156)—a growing group of volunteers who regularly contribute to Learning Equality's repositories. It’s a great way to connect, with an option to have your contributions featured on social media platforms like LinkedIn._**

**Comment by Suraj-kumar00** [comment]
Thanks @MisRob, It was an great experience to work with kolibri. Learned a lot.

**Reply by MisRob** [reply]
We're glad to hear that @Suraj-kumar00, thank you

**Comment by rtibbles** [comment]
Thank you to all our contributors - you make a big difference to our project!
